src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -118,7 +118,7 @@ static ExperimentalFeature()
                     description: "Replace the hard-coded suggestion framework with the extensible feedback provider"),
                 new ExperimentalFeature(
                     name: PSNativeWindowsTildeExpansion,
-                    description: "On windows, expand unquoted tilde (`~`) with the user's current home folder."),
```

Comment:
With lowercase, it my imply and invoke the GUI windows, which doesn't make much sense.